### ISSUE TITLE
Reorganize Examples

### ISSUE THREAD

**Issue by pathunstrom** [ppb/pursuedpybear#320]
I don't like the current layout for examples. It's a confusing bundle of script files that don't really demo best practices. Proposing a modification to the structure:

```
examples
- {game 1}
-- main.py
-- resources
---- {image files}
- {game 2}
-- setup.py
-- {packaged game}
---- resources
---- __main__.py
```

The goal is to make each folder standalone and usable as a skeleton to start a new project off of. We don't actually package examples and they live on github, so the extra files aren't likely to cause issues.

**Comment by pathunstrom** [comment]
I don't expect this to be done all at once, this is a thing that can be broken down into individual issues for each example and done by newer contributors. This is a proposal for "what we expect".

**Comment by pathunstrom** [comment]
Closed via #333
